### DISCUSSION TITLE
System Requirements Specification (SRS) document

### DISCUSSION THREAD

**Discussion by TemirlanAidarov** [sequelize/sequelize#14166] [Help & Questions] [answered]
Hello!
I have just found about this project.
I am Computer Science student who takes a course related to Open Software Project.
We have an assignment to write/improve System Requirements Specification (SRS) document for any project. Basically, the SRS is a document that describes the nature of a project, software or application. In simple words, SRS document is a manual of a project provided it is prepared before you kick-start a project/application.

Can I write/improve SRS document for this project?

**Comment by WikiRik** [comment, marked as answer]
Hi! Thanks for your message. We're happy to see that we're one of the projects you are considering for this, but seeing that we've been working on this project for about 10 years already there are quite some specifications to this project.
I think that @sdepold can best answer your question however so I'll leave it up to him
